onboarding.html

```diff
@@ -30,17 +30,17 @@
       <div id="page1" class="text-section">
         <h1>reflect.</h1>
         <h2>welcome home!</h2>
-        <p>to use reflect, type your reason for visiting the website in the text box. to change your list of distracting websites, press “options.”</p>
+        <p>to use reflect, type your reason for visiting the website in the text box. the extension will return a response after you press enter”</p>
         <a id="page1button" class="lt-hover white_button shadow">got it!</a>
       </div>
       <div id="page2" class="text-section">
-        <h2>options.</h2>
-        <p>on this page, you can add distracting websites or change how long you’re allowed into those website for. you can also get here by right-clicking the reflect icon and selecting 'show filter list.'</p>
+        <h2>on and off.</h2>
+        <p>to turn the extension off for longer breaks, press the icon in the upper right of your chrome menu to open the toggle menu.</p>
         <a id="page2button" class="lt-hover white_button shadow">next.</a>
       </div>
       <div id="page3" class="text-section">
-        <h2>on and off.</h2>
-        <p>to turn the extension off for longer breaks, press the icon in the upper right of your chrome menu.</p>
+        <h2>options.</h2>
+        <p>on this page, you can add distracting websites or change how long you’re allowed into those website for. you can also view your latest intents and set how many intents you want to display.</p>
```

Comment:
maybe rephrase so that main purpose of options page is to customize your experience with reflect, then talk about what else it does

Reply:
ooo i like this